plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/BaseKuduConnectorSmokeTest.java

```diff
@@ -150,4 +150,20 @@ public void testUpdate()
                 .matches(expectedValues("(0, 1.2), (1, 2.5), (2, 6.2), (3, 7.5), (4, 11.2)"));
         assertUpdate("DROP TABLE " + tableName);
     }
+
+    @Test
+    public void testCreateTableWithTableComment()
+    {
+        String tableName = "test_create_" + randomNameSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " (a bigint WITH (primary_key=true)) COMMENT 'test comment' WITH (partition_by_hash_columns = ARRAY['a'], partition_by_hash_buckets = 2)");
+
+        if (getKuduServerVersion().equals(EARLIEST_TAG)) {
+            assertNull(getTableComment(tableName));
```

Comment:
Could not find a way to throw an exception when user attempts to add table comment on EARLIEST_TAG. If we don't want to fail silently, we will need to figure out a way to check kudu version at runtime?